.github/workflows/ci_workflows.yml

```
@@ -56,47 +56,37 @@ jobs:
 
           # NOTE: this 2nd coverage test is needed for tests and code that
           #       run only with minimal dependencies.
-          - name: Python 3.9 with minimal dependencies and full coverage
-            os: ubuntu-latest
-            python: 3.9
-            toxenv: py39-test-cov
-
-          - name: Python 3.10 with minimal dependencies
+          - name: Python 3.10 with minimal dependencies and full coverage
             os: ubuntu-latest
             python: '3.10.0-alpha - 3.10.0'
```

We no more need the complex conditions, also I'm not sure the current one would pick bufigx releases.
```suggestion
            python: '3.10'
```

---

Is this available yet? I tried this for a different project just 1-2 weeks ago and it failed.

---

Yes, 3.10 is available, I tested recently on astroscrappy.